libraries/engage/core/selectors/shopSettings.js

```diff
@@ -0,0 +1,105 @@
+import { createSelector } from 'reselect';
```

Comment:
The shop settings feature does not exist in pwa 6 and probably will never exist there. So please remove this.

Reply:
done